app/config/themes/default.json

```diff
@@ -0,0 +1,67 @@
+{
```

Comment:
@akariv 

All looks good, just to clarify:

1. The themes can't be injected from any old location - they need to actually be in the install. This seems like a good decision from a security perspective, especially as the theming API allows setting text and links, but I just want to make sure that is now the intention (not allow remote injection of theming colors, etc.)
2. Due to the complexities of theming, I guess you choose a JSON object over env vars, which seems reasonable. I just wonder if for custom installs, an env var that points to an additional theme directory, or a theme file, might also be useful, and prevent for example, docker ADD or other server actions on custom installs.